### DISCUSSION TITLE
Iframe break the JavaScript

### DISCUSSION THREAD

**Discussion by Mohamed-Kaizen** [mkdocs/mkdocs#3003] [Q&A] [answered]
when adding an iframe it will break the JS on the site, like search, copy to clip board, etc...

[Reproduction](https://github.com/Mohamed-Kaizen/mkdocs-material-iframe-issue-reproduction)

[Live Demo](https://mkdocs-material-iframe-issue.netlify.app/)

And yes i did check the error is not from `mkdocs-material` b/c when i removed the theme the default search of mkdocs is not working

**Comment by oprypin** [comment, marked as answer]
`<iframe whatever=whatever/>  other stuff`
is the same as
`<iframe whatever=whatever>  other stuff`

but you need to write
`<iframe whatever=whatever></iframe>  other stuff`

otherwise the rest of the page's code will be eaten by the iframe

as can be seen from syntax highlighting

![image](https://user-images.githubusercontent.com/371383/195721635-b08710ac-f2aa-4c40-8abe-0f3b4c737f9b.png)

Thanks for the live demo

**Comment by Mohamed-Kaizen** [comment]
I got where i make the mistake, Thanks
